src/App.jsx

```diff
@@ -2,15 +2,15 @@ import "./App.css";
 import Header from "./components/Header";
 import Feed from "./components/Feed";
 import { Routes, Route } from "react-router-dom";
-import AllArticles from "./components/AllArticles";
+import FullArticle from "./components/FullArticle";
 
 function App() {
   return (
     <>
       <Header />
       <Routes>
         <Route path="/" element={<Feed />} />
-        <Route path="/all-articles" element={<AllArticles />} />
+        <Route path="/full-article/:article_id" element={<FullArticle />} />
```

Comment:
as this is what the user sees, it would be better to have this path as /articles/:article_id - but just a suggestion :)